pkg/runtime/runtime_test.go

```diff
@@ -1267,70 +1267,266 @@ func TestMetadataNamespace(t *testing.T) {
 	assert.NoError(t, err)
 }
 
-func TestMetadataClientID(t *testing.T) {
```

Comment:
Why did this test get deleted?

Reply:
it was renamed to the below: `TestKafkaMetadataClientID`